integreat_cms/cms/views/pois/poi_form_view.py

```diff
@@ -177,6 +178,25 @@ def post(self, request, *args, **kwargs):
                     language__in=languages
                 ).update(status=status.DRAFT)
 
+            # Show a message that the slug was changed if it was not unique
+            if user_slug and user_slug != poi_translation_form.cleaned_data["slug"]:
+                other_translation = POITranslation.objects.filter(
+                    poi__region=region, slug=user_slug, language=language
+                ).first()
+                other_translation_link = other_translation.backend_edit_link
+                messages.warning(
+                    request,
+                    _(
+                        "The slug was changed from '%(user_slug)s' to '%(slug)s', because '%(user_slug)s' is already used by <a href='%(link)s' class='underline hover:no-underline'>%(translation)s</a> or one of its previous versions"
+                    )
+                    % {
+                        "user_slug": user_slug,
+                        "slug": poi_translation_form.cleaned_data["slug"],
+                        "link": other_translation_link,
+                        "translation": other_translation,
+                    },
```

Comment:
```suggestion
                        "The slug was changed from '{user_slug}' to '{slug}', because '{user_slug}' is already used by <a href='{link}' class='underline hover:no-underline'>{translation}</a> or one of its previous versions"
                    ).format(
                        user_slug=user_slug,
                        slug=poi_translation_form.cleaned_data["slug"],
                        link=other_translation_link,
                        translation=other_translation,
                    ),
```